parsers/nmparser/parse_run_details.go

```diff
@@ -42,21 +42,21 @@ func parseLine(line string, n int) string {
 
 // ParseRunDetails parses run details such as start date/time and estimation time etc.
 func ParseRunDetails(lines []string) RunDetails {
-	version := ""
-	runStart := ""
-	runEnd := ""
-	estimationTime := 0.0
-	covarianceTime := 0.0
-	functionEvaluations := int64(0)
-	significantDigits := 0.0
-	problemText := ""
-	modFile := "" // TODO, pass in from caller
+	version := DefaultString
```

Comment:
per style guide: https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations

this should be grouped and use a var( declaration)

```
var (
 runStart = DefaultString
 runEnd = DefaultString
...
) 
```

and should be organized by type

Reply:
at this point, it might make more sense anyway just to initialize RunDetails with NewRunDetails() then set things on that directly and completely get rid of all these variables

Reply:
I agree, I was very close to doing that ...